content/docs/guides/promo-expressions.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Rules-Based Promotion Expressions'
+section: 'Guides'
+priority: 4
+hidden: false
+---
+
+Writing custom rule-base expressions for your order promotions may seem intimidating at first, but once you get the hang of it you'll realize there are virtually limitless possibilities which provide the flexibility to meet almost any business need. In fact, custom order promotions are a great example of how our core value *Flexibility over Features* informs everything we build (and don't build) into the platform. We've found that allowing developers to define their own custom if-then statements is incredibly robust and solves more problems than providing a limited set of non-customizable promotions out-of-the-box.
+
+## Where should I start?
+
+Every promotion requires two rule expressions, an `EligibleExpression`, which evaluates the current state of the order and returns true or false indicating whether the promotion can be applied to the order, and a `ValueExpression`, which evaluates the order and returns a monetary value, which is then subtracted from the order subtotal.
+
+**There are several properties that you can include in both expressions to build your custom promotion:**\
+`order` supports the same properties as the Order model returned from /orders API endpoints, including xp.\
+`items` supports a total of four functions:
+ - `items.any()` (true if any item on the order matches filter)
+ - `items.all()` (true if all item matches filter)
+ - `items.quantity()`(compare result to a number)
+ - `items.total()` (compare result to a dollar amount)
+
+**The following operators are all supported in rule engine expressions:**
+-  Comparison: `=` ,  `<` ,  `>`,  `<=`,  `>=`  
+-  Logical: `and`,  `or`  and  `not`
+-  Mathematical:`+`,  `-`,`*`, `/` and `%` 
+
+##  Can you give me some examples?
+
+**$10 off any order greater than $50:**\
+`EligibleExpression`: *"order.Subtotal > 50"*\
+`ValueExpression`: *"10"*
```

Comment:
Doesn't look like this is rendering how you'd expect it. Line 29, 30 and 31 show up as a single line with the backslash present. Did a little digging and it looks like Github's flavor of markdown interprets a backslash at the end of a line as a hard break but vanilla markdown does not. Luckily I found you can get the same behavior by ending the line with two spaces. Still not ideal though, I prefer github's syntax because it makes it more explicit

Reply:
okay I'll switch to the syntax in that example you sent me!